articles/quickstart/native/android/_includes/_auth0.md

```diff
@@ -19,10 +19,12 @@ dependencies {
 ```
 
 ::: note
-You can check for the latest version on the repository [Readme](https://github.com/auth0/auth0.android#installation), in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22), or in [JCenter](https://bintray.com/auth0/android/auth0).
+You can check for the latest version on the [repository Readme](https://github.com/auth0/auth0.android#installation), in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22), or in [JCenter](https://bintray.com/auth0/android/auth0).
 :::
 
-Now add the _Manifest Placeholders_, required by the SDK to define internally an **intent-filter** to capture the authentication callback. You do that by adding the next line:
+Add manifest placeholders required by the SDK. The placeholders define internally an `intent-filter` to capture the authentication callback URL. 
```

Comment:
placeholders are used to generate the intent-filter. (the library generates it). I would use:
"Add manifest placeholders required by the SDK. Placeholders are used internally to define an `intent-filter` that captures the authentication callback URL."